docs/get_started_with_qiskit.ipynb

```diff
@@ -0,0 +1,304 @@
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "89f42d59-305c-4d6d-b6d4-c300afc91de6",
+   "metadata": {
+    "editable": true,
+    "raw_mimetype": "text/restructuredtext",
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "This tutorial uses Qiskit's built-in :class:`~.primitives.Sampler`, which simulates a perfect quantum computer, but can only handle small numbers of qubits. You can replace this with samplers of higher-performance simulators from `Qiskit Aer <https://github.com/Qiskit/qiskit-aer#readme>`__, or with samplers that send your circuit over the internet to run on real quantum computers (such as `IBM's Qiskit Runtime <https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime/runtime_service>`__ sampler)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "69a8d872",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "SamplerResult(quasi_dists=[{0: 0.4999999999999999, 3: 0.4999999999999999}], metadata=[{}])"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.primitives import Sampler\n",
+    "\n",
+    "# Create a Sampler object\n",
+    "sampler = Sampler()\n",
+    "\n",
+    "# Submit the circuit to the sampler\n",
+    "job = sampler.run(qc)\n",
+    "\n",
+    "# Once the job is complete, get the result\n",
+    "job.result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc5ce1eb",
+   "metadata": {},
+   "source": [
+    "The `quasi_dists` property is a list of sampled distributions for each circuit we provided. Since we only provided one circuit, the list only has one element.\n",
+    "\n",
+    "The distributions are dictionary-like objects. Each key is a measured output, and its value is the fraction of the time that output was measured. In this case, the circuit measurement output was `0` ~50% of the time, and `3` (which is `11` in binary) ~50% of the time."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b687ce04-281c-4ca1-89f1-90771ad192e4",
+   "metadata": {},
+   "source": [
+    "## Plotting results"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "d02639fa-ff92-4155-9b36-07112bbf991d",
+   "metadata": {
+    "editable": true,
+    "raw_mimetype": "text/restructuredtext",
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "You can use the :func:`~.visualization.plot_histogram` function to plot your `Sampler` results. This function returns a MatPlotLib `Figure`, which displays nicely in Jupyter notebooks. For more ways to display results, see `Qiskit visualizations <https://qiskit.org/documentation/tutorials/circuits/2_plotting_data_in_qiskit.html>`__."
```

Comment:
If we're in rst space can we use references instead of external links? If we can that'll be a bit more robust for different builds of the documentation.

Reply:
I don't think this is possible until #10710 is merged